jsk_fetch_robot/jsk_fetch.rosinstall.kinetic

```diff
@@ -0,0 +1,37 @@
+- git:
+    local-name: jsk-ros-pkg/jsk_recognition
+    uri: https://github.com/jsk-ros-pkg/jsk_recognition
+    version: 1.2.2
+- git:
+    local-name: jsk-ros-pkg/jsk_pr2eus
+    uri: https://github.com/jsk-ros-pkg/jsk_pr2eus.git
+    version: 0.3.7
```

Comment:
latest version is 0.3.10, why 0.3.7
http://repositories.ros.org/status_page/ros_kinetic_default.html?q=jsk_pr2eus